tidy3d/__init__.py

```diff
@@ -3,14 +3,6 @@
 
 from rich import pretty, traceback
 
-from .config import config
```

Comment:
so we can add config back in now that web is removed?

Reply:
It's there, just put it lower in the init file. There is a strange circular import which seems to not be an issue depending on the order of the imports. Namely updater imports from tidy3d.components, which causes it to import the whole module, which imports Simulation, but Simulation imports updater. When I removed config initially, this created circular import (probably something to do with components being initialized in config already before updater is imported?) Anyway I decided to put everything after all the components initializations. I don't know if there's a bigger problem here or not, but generally some examples like that have made it clear to me that there are consequences to importing too many things in a module's `__init__`...